src/typings/dcos__http-service.d.ts

```diff
@@ -1,13 +1,24 @@
 import { Observable } from "rxjs";
 
 interface RequestOptions {
-  method?: string
-  body?: any
-  headers?: Object
-  responseType?: string
-  timeout?: number
+  method?: string;
+  body?: any;
+  headers?: Object;
+  responseType?: string;
+  timeout?: number;
 }
 
-export function request<T>(url: string, options?: RequestOptions): Observable<T>;
-export function stream<T>(url: string, options?: RequestOptions): Observable<T>;
+export interface RequestResponse<T> {
+  code: number;
+  message: string;
+  response: T;
+}
 
+export function request<T>(
+  url: string,
+  options?: RequestOptions
+): Observable<RequestResponse<T>>;
+export function stream<T>(
```

Comment:
But does stream return `RequestResponse`?

Reply:
No it doesn't, fixed the typo